CHANGELOG.rst

```diff
@@ -5,13 +5,27 @@
 
 Renku ``0.21.0`` brings the ability to handle expiring Gitlab tokens in user sessions and various bug fixes. 
 
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**✨ Improvements**
+
+* 🎨 **UI**: Improve layout on different pages: project creation, starting sessions, file browser.
+
+**🐞 Bug Fixes**
+
+* 📃 **UI**: Update broken links local projects, to documentation, and add more links to useful resources.
+* 🚀 **UI**: Fix glitches with autosave and improve layout of session pages.
+* 🔦 **UI**: Prevent flashing inputs when forking a project.
+
 Internal Changes
 ~~~~~~~~~~~~~~~~
 
 **Improvements**
 
 * **Gateway**: Add endpoint for refreshing expiring Gitlab tokens
 * **Sessions**: Check LFS size and available disk space before cloning
+* **UI**: Receive notifications when sessions change
```

Comment:
`Added support for websocket-based notifications when session state changes`

Reply:
I would like to keep the changelog as non-technical as possible, leaving out the WebSocket detail.

Reply:
I agree with Sean. This is already in the "non-user facing" part of the changelog, where the goal of the changelog is primarily so we (the development team) can keep track of and refer to what was released when. We should make the descriptions specific enough so we remember what the point is referring to.